src/components/theme/ContactInfo/ContactInfo.jsx

```diff
@@ -0,0 +1,238 @@
+/**
+ * Login container.
```

Comment:
@cekk you forgot to amend the comments here. Should be "ContactInfo" not "Login"